components/OrganizationInfo.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div class="info-container">
+      <div class="info-items">
+          <h2>団体情報</h2>
+          <table class="info-list">
+              <tr>
+                  <th>団体名</th>
+                  <td>特定非営利活動法人アイセック・ジャパン （AIESEC in Japan）</td>
+              </tr>
+              <tr>
+                  <th>所轄庁</th>
+                  <td>東京都</td>
+              </tr>
+              <tr>
+                  <th>設立</th>
+                  <td>1962年 （2001年　特定非営利活動法人格取得）</td>
+              </tr>
+              <tr>
+                  <th>代表</th>
+                  <td>各務茂夫 （東京大学 大学院工学系研究科教授 産学協創推進本部 副本部長）</td>
+              </tr>
+              <tr>
+                  <th>専務理事</th>
+                  <td>森原正希 （早稲田大学創造理工学部建築学科3年）</td>
+              </tr>
+              <tr>
+                  <th>所在地</th>
+                  <td>〒162-0814 東京都新宿区新小川町4-16飯田橋プラレールビル3F</td>
+              </tr>
+              <tr>
+                  <th>活動内容</th>
+                  <td>
+                        社会教育の推進を図る活動<br>
+                        国際協力の活動<br>
+                        前各号の掲げる活動を行う団体の運営又は活動に関する連絡、助言又は
+                        援助の活動
```

Comment:
```suggestion
                        前各号の掲げる活動を行う団体の運営又は活動に関する連絡、助言又は援助の活動
```
変なスペースが入ってしまうので改行消してください🙇‍♂️